scripting/include/smlib/clients.inc

```diff
@@ -585,9 +585,9 @@ enum Water_level
  * @param client		Client index.
  * @return				Returns 0 if not in water. 1 if feets are in water. 2 if waist is in water. 3 if head is in water.
  */
-stock Water_Level:Client_GetWaterLevel(client)
+stock int Client_GetWaterLevel(int client)

```

Comment:
I have an error : 

`F:\___Servers\_compile\1.9.0_6014\include\smlib/clients.inc(585) : error 139: could not find type "Water_Level"`

that's why i put it `int`